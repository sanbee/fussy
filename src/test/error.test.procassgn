f(x){printn x;}
y=f(10.0);
y:=f(10pm1);
