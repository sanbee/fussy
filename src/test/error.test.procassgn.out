   fussy 2.0
   Copyright (c) 2000-2017, 2018 S.Bhatnagar (sbhatnag (AT) nrao (DOT) edu)
   This is free software with ABSOLUTELY NO WARRANTY.
   For details type `warranty'.

###Error: Syntax error near token 'f' in "y=f(10.0);"
###Error: PROC used in an assigment ('=') statement
